.travis.yml

```diff
@@ -63,12 +63,12 @@ matrix:
         - pip install bump2version
       script:
         - set -e
-        - git pull --no-edit origin master
+        - git pull --no-edit origin main
```

Comment:
This command is going to print a warning. You need some `git config` here.
```
git config --global pull.rebase false
```

Reply:
Thanks! good to know :)